cmd/promql-langserver/promql-langserver.go

```diff
@@ -44,7 +45,18 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		handler, err := rest.CreateInstHandler(context.Background(), prometheusClient, prometheus.NewRegistry())
+
+		var logger kitlog.Logger
+
+		if config.LogFormat == "json" {
```

Comment:
Here you Can use the enum const from the config. Like that we won't forget some part if this value is changing